.github/Git workflow.md

```diff
@@ -0,0 +1,67 @@
+0. Don't make changes to `master` branch.
```

Comment:
Suggestion: avoid using space in file name. Consider using `-` all  with lower case